docs/src/index.rst

```diff
@@ -1,9 +1,9 @@
-.. Orion documentation master file, created by
+.. Oríon documentation master file, created by
    sphinx-quickstart on Mon Nov 13 19:08:05 2017.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Orion's documentation!
+Welcome to Oríon's documentation!
 ===================================
```

Comment:
I missed this underline that's too long

Reply:
Don't worry about it. I am changing it completely in #59